BlockServerToKafka/main.py

```diff
@@ -16,22 +16,21 @@
 
 import os
 import sys
+sys.path.insert(0, os.path.abspath(os.getcwd()))
 from argparse import ArgumentParser
 from BlockServerToKafka.block_server_monitor import BlockServerMonitor
 from time import sleep
 from os import environ
 from BlockServerToKafka.kafka_producer import Producer
 
-sys.path.insert(0, os.path.abspath(os.getcwd()))
-
 if __name__ == '__main__':
     parser = ArgumentParser()
 
     parser.add_argument('-d', '--data', help='Kafka topic to send PV data to', nargs=1, type=str,
                         default='test_bs_forwarder')
     parser.add_argument('-c', '--config', help='Kafka topic to send forwarder config to', nargs=1, type=str,
                         default='test_bs_forwarder_config')
-    parser.add_argument('-b', '--broker', help='Location of the Kafka brokers (host:port)', nargs=1, type=str,
+    parser.add_argument('-b', '--broker', help='Location of the Kafka brokers (host:port)', nargs='*', type=str,
```

Comment:
Probably better to use + here rather than * as + means you have to give at least one broker.